src/Tizen.Applications.Alarm/Tizen.Applications/AlarmManager.cs

```diff
@@ -74,6 +74,34 @@ public enum AlarmWeekFlag
         WeekDays = Monday | Tuesday | Wednesday | Thursday | Friday
     }
 
+    /// <summary>
+    /// Enumeration for inexact interval.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    [Flags]
+    public enum AlarmStandardPeriodFlag
```

Comment:
DO NOT mark an enumeration with FlagsAttribute if it has one or more values that are not powers of two.